.storybook/theme.ts

```diff
@@ -0,0 +1,31 @@
+import { create } from "@storybook/theming";
+import { JobberStyle } from "@jobber/design/foundation.js";
+
+// eslint-disable-next-line import/no-default-export
+export default create({
+  base: "light",
+
+  /**
+   * Fonts
+   */
+  fontBase: JobberStyle["typography--fontFamily-normal"],
```

Comment:
Do they have a concept of a display/heading font in Storybook?

Reply:
they do not. we'd have to inject jobber components in those things. Also... i think that's the reason why the header links doesn't exist on mobile. 😆